pkg/machine/e2e/basic_test.go

```diff
@@ -22,27 +22,27 @@ var _ = Describe("run basic podman commands", func() {
 	It("Basic ops", func() {
 		name := randomString(12)
 		i := new(initMachine)
-		session, err := mb.setName(name).setCmd(i.withImagePath(mb.imagePath).withNow()).run()
+		session, err := mb.setName(name).setCmd(i.withImagePath(mb.imagePath).withNow()).run(true)
```

Comment:
Can you just add another function. The diff is unnecessary large because of this. It is also very annoying when you write new test to always have to set it.

Just keep run() and add another function runWithoutWait() to do what you need.